release-notes-5.0.0.md

```diff
@@ -0,0 +1,14 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### New Features
```

Comment:
You're missing the new feature here that the workflows can now contain Graal.js scripts (in addition to the Nashorn scripts which continue to be supported).